ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+75.8 "Crumbling Crust (cast)"
+77.3 "Crumbling Crust (circles)"
+89.0 "Rush"
+91.1 "War Cry"
+93.8 "Earthquake"
+
+#~~~~~~~~#
+# BARDAM #
+#~~~~~~~~#
+
+# -ii 1EFE 2578 2579 257A 257E 257B
+
+1000.0 "Start" sync / 00:0839:.*Rebirth of Bardam the Brave will be sealed off/ window 1000,5
+1007.8 "Travail" sync /:Bardam:1EFF:/ window 7.8,5
+1014.1 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1020.3 "Tremblor" sync /:Bardam:257C:/
+1022.8 "Empty Gaze" sync /:Hunter Of Bardam:1F04:/
+1025.0 "Travail" sync /:Bardam:1EFF:/
+1029.8 "Charge x3" # sync /:Throwing Spear:257F:/
+1038.8 "Empty Gaze" sync /:Hunter Of Bardam:1F04:/
+1038.9 "Charge" sync /:Throwing Spear:257F:/
+1044.6 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1052.3 "Travail" sync /:Bardam:1EFF:/
+1060.5 "Sacrifice" sync /:Bardam:1F01:/
+1060.7 "Travail" sync /:Bardam:1EFF:/
+1065.5 "Bardam's Ring" sync /:Bardam:2581:/
+1066.2 "Travail" sync /:Bardam:1EFF:/
+1076.4 "Comet x8" duration 12 # sync /:Bardam:257D:/
+1093.4 "Heavy Strike x3" sync /:Hunter Of Bardam:2577:/ duration 4
+1099.0 "Travail" sync /:Bardam:1EFF:/
+1105.3 "Comet Impact" sync /:Star Shard:2580:/
+1109.3 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1116.9 "Travail" sync /:Bardam:1EFF:/
+1125.3 "Reconstruct" sync /:Bardam:1EFD:/
+1126.7 "--sync--" sync /:Star Shard:258B:/
+1128.3 "Travail" sync /:Bardam:1EFF:/
+1133.1 "Charge" sync /:Throwing Spear:257F:/
+1133.9 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1139.5 "Tremblor" sync /:Hunter Of Bardam:2585:/
+1140.0 "Heavy Strike x3" sync /:Warrior Of Bardam:2577:/ duration 4
+1153.0 "Meteor Impact" sync /:Looming Shadow:2582:/
+1153.4 "--sync--" sync /:Star Shard:258B:/
+
+#~~~~~#
+# YOL #
+#~~~~~#
+
+# -ii 1F0B 1F0E
+
+2000.0 "Start" sync / 00:0839:.*Voiceless Muse will be sealed off/ window 2000,5
+2008.1 "Feathercut" sync /:Yol:1F09:/
+2015.2 "Wind Unbound" sync /:Yol:1F0A:/
+2023.9 "Pinion" sync /:Yol Feather:1F11:/
+2025.8 "Flutterfall" sync /:Yol:1F0C:/
+2027.2 "Feathercut" sync /:Yol:1F09:/
+2033.9 "Pinion" sync /:Yol Feather:1F11:/
+2034.7 "Flutterfall" sync /:Yol:1F0C:/
+2042.2 "Wind Unbound" sync /:Yol:1F0A:/
+2046.4 "Feathercut" sync /:Yol:1F09:/
+2050.9 "Pinion" sync /:Yol Feather:1F11:/
+2054.0 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2059.2 "Feathercut" sync /:Yol:1F09:/
+2060.9 "Pinion" sync /:Yol Feather:1F11:/
+2069.3 "Wind Unbound" sync /:Yol:1F0A:/
+2071.4 "--untargetable--"
+2071.4 "--adds spawn--" sync /03:........:Added new combatant Corpsecleaner Eagle/ window 71.4,5
+2078.0 "Pinion" sync /:Yol Feather:1F11:/
+2088.0 "Pinion" sync /:Yol Feather:1F11:/
+2098.0 "Pinion" sync /:Yol Feather:1F11:/
+2108.0 "Pinion" sync /:Yol Feather:1F11:/
+2118.0 "Pinion" sync /:Yol Feather:1F11:/ jump 2078.0
+2128.0 "Pinion"
+2138.0 "Pinion"
+2148.0 "Pinion"
+
+# Unfortunately, because there are two Corpsecleaner Eagles,
+# we can't just sync to a 04 log line.
+2189.7 "--targetable--" sync /22:........:Yol:........:Yol:01/ window 120,5
+2196.8 "Feathercut" sync /:Yol:1F09:/
```

Comment:
It might be worth having a larger sync here to satisfy test_timeline / fflogs which don't have 22 line support.

Reply:
Implementing this and running against a few timelines I have, it looks like it's probably okay. There are a couple of syncs after the Eagle intermission that swing by about 0.1s, and the Eagle intermission itself can vary by about 0.5s, but other than that everything seems to be under 0.1s variance.

```
0.000: Matched entry: 2000.0 Start (+2000.000s)
2008.107: Matched entry: 2008.1 Feathercut (-0.007s)
2015.213: Matched entry: 2015.2 Wind Unbound (-0.013s)
2023.910: Matched entry: 2023.9 Pinion (-0.010s)
2025.773: Matched entry: 2025.8 Flutterfall (+0.027s)
2027.223: Matched entry: 2027.2 Feathercut (-0.023s)
2033.880: Matched entry: 2033.9 Pinion (+0.020s)
2034.704: Matched entry: 2034.7 Flutterfall (-0.004s)
2042.155: Matched entry: 2042.2 Wind Unbound (+0.045s)
2046.356: Matched entry: 2046.4 Feathercut (+0.044s)
2050.908: Matched entry: 2050.9 Pinion (-0.008s)
2053.982: Matched entry: 2054.0 Eye Of The Fierce (+0.018s)
2059.174: Matched entry: 2059.2 Feathercut (+0.026s)
2060.939: Matched entry: 2060.9 Pinion (-0.039s)
2069.322: Matched entry: 2069.3 Wind Unbound (-0.022s)
2072.166: Matched entry: 2071.4 --adds spawn-- (-0.766s)
2077.268: Matched entry: 2078.0 Pinion (+0.732s)
2087.960: Matched entry: 2088.0 Pinion (+0.040s)
2098.030: Matched entry: 2098.0 Pinion (-0.030s)
2107.991: Matched entry: 2108.0 Pinion (+0.009s)
2118.014: Matched entry: 2118.0 Pinion (-0.014s)
    Jumping to 2078.000
2078.000: Matched entry: 2078.0 Pinion (+0.000s)
2088.051: Matched entry: 2088.0 Pinion (-0.051s)
2098.001: Matched entry: 2098.0 Pinion (-0.001s)
2108.011: Matched entry: 2108.0 Pinion (-0.011s)
2118.004: Matched entry: 2118.0 Pinion (-0.004s)
    Jumping to 2078.000
2078.000: Matched entry: 2078.0 Pinion (+0.000s)
2088.042: Matched entry: 2088.0 Pinion (-0.042s)
2098.021: Matched entry: 2098.0 Pinion (-0.021s)
2108.036: Matched entry: 2108.0 Pinion (-0.036s)
2116.743: Matched entry: 2196.8 Feathercut (+80.057s)
    Missed sync: Pinion at 2118.0 (last seen at 2108.0)
    Missed sync: --targetable-- at 2189.7
2203.966: Matched entry: 2204.0 Wind Unbound (+0.034s)
2212.727: Matched entry: 2212.7 Pinion (-0.027s)
2214.525: Matched entry: 2214.5 Flutterfall (-0.025s)
2215.834: Matched entry: 2215.8 Feathercut (-0.034s)
2222.610: Matched entry: 2222.6 Pinion (-0.010s)
2223.266: Matched entry: 2223.3 Flutterfall (+0.034s)
2230.682: Matched entry: 2230.7 Wind Unbound (+0.018s)
2234.884: Matched entry: 2234.9 Feathercut (+0.016s)
2239.443: Matched entry: 2239.4 Pinion (-0.043s)
2242.471: Matched entry: 2242.5 Eye Of The Fierce (+0.029s)
2247.666: Matched entry: 2247.7 Feathercut (+0.034s)
2249.442: Matched entry: 2249.4 Pinion (-0.042s)
2260.272: Matched entry: 2260.3 Flutterfall (+0.028s)
    Missed sync: --untargetable-- at 2253.8
2267.565: Matched entry: 2267.5 Wingbeat (-0.065s)
2276.980: Matched entry: 2276.9 Flutterfall (-0.080s)
2284.335: Matched entry: 2284.3 Wingbeat (-0.035s)
2284.746: Matched entry: 2284.7 Pinion (-0.046s)
2293.734: Matched entry: 2293.8 Flutterfall (+0.066s)
2301.229: Matched entry: 2301.2 Wingbeat (-0.029s)
2301.644: Matched entry: 2301.6 Pinion (-0.044s)
2310.707: Matched entry: 2310.7 Flutterfall (-0.007s)
2318.217: Matched entry: 2318.2 Wingbeat (-0.017s)
2318.555: Matched entry: 2318.6 Pinion (+0.045s)
    Jumping to 2284.700
2293.829: Matched entry: 2293.8 Flutterfall (-0.029s)
2301.300: Matched entry: 2301.2 Wingbeat (-0.100s)
2301.557: Matched entry: 2301.6 Pinion (+0.043s)
2310.707: Matched entry: 2310.7 Flutterfall (-0.007s)
2318.167: Matched entry: 2318.2 Wingbeat (+0.033s)
2318.600: Matched entry: 2318.6 Pinion (+0.000s)
    Jumping to 2284.700
2301.038: Matched entry: 2368.8 Feathercut (+67.762s)
    Missed sync: Flutterfall at 2293.8 (last seen at 2310.707)
    Missed sync: Wingbeat at 2301.2 (last seen at 2318.167)
    Missed sync: Pinion at 2301.6 (last seen at 2318.6)
    Missed sync: Wingbeat at 2318.2 (last seen at 2301.3)
    Missed sync: Pinion at 2318.6 (last seen at 2301.557)
    Missed sync: --targetable-- at 2361.7
2376.003: Matched entry: 2376.0 Wind Unbound (-0.003s)
2384.716: Matched entry: 2384.7 Pinion (-0.016s)
2386.564: Matched entry: 2386.6 Flutterfall (+0.036s)
2394.722: Matched entry: 2394.7 Pinion (-0.022s)
2395.454: Matched entry: 2395.5 Flutterfall (+0.046s)
2402.897: Matched entry: 2402.9 Wind Unbound (+0.003s)
2407.112: Matched entry: 2407.1 Feathercut (-0.012s)
2411.582: Matched entry: 2411.6 Pinion (+0.018s)
2414.749: Matched entry: 2414.7 Eye Of The Fierce (-0.049s)
2419.885: Matched entry: 2419.9 Feathercut (+0.015s)
2421.542: Matched entry: 2421.5 Pinion (-0.042s)
2433.090: Matched entry: 2433.1 Feathercut (+0.010s)
2440.228: Matched entry: 2440.2 Wind Unbound (-0.028s)
2448.946: Matched entry: 2448.9 Pinion (-0.046s)
2450.680: Matched entry: 2450.7 Flutterfall (+0.020s)
2452.081: Matched entry: 2452.1 Feathercut (+0.019s)
2458.906: Matched entry: 2458.9 Pinion (-0.006s)
2459.566: Matched entry: 2459.6 Flutterfall (+0.034s)
2467.040: Matched entry: 2467.0 Wind Unbound (-0.040s)
2471.138: Matched entry: 2471.1 Feathercut (-0.038s)
2475.687: Matched entry: 2475.7 Pinion (+0.013s)
2478.685: Matched entry: 2478.7 Eye Of The Fierce (+0.015s)
2483.877: Matched entry: 2483.9 Feathercut (+0.023s)
2485.726: Matched entry: 2485.7 Pinion (-0.026s)
```